combat-trainer.lic

```diff
@@ -65,6 +70,27 @@ class SetupProcess
 
   private
 
+  def blow_warhorn()
+    return unless @warhorn
+    return if Time.now - @last_warhorn < 300
+
+    if DRCI.wearing?(@warhorn)
+        get_verb = "remove"
+        store_verb = "wear"
+    else
+        get_verb = "get"
+        store_verb = "stow"
+    end
+
+    case bput("#{get_verb} my #{@warhorn}", 'You get', 'You remove', 'What were you referring to', 'You need a free hand', 'Remove what')
```

Comment:
What do you think about a message when no warhorn is found?

```
when 'What were you referring to'
  echo "#{@warhorn} NOT FOUND! Removing from hunt."
  @warhorn = ''
end
```

This would stop it from tapping each time through again and save a little time triggering the `return unless @warhorn` before the tap.

Reply:
done